tools/train_ensemble.py

```diff
@@ -0,0 +1,112 @@
+"""Anomalib Training Script for ensemble of models.
+
+This script reads the name of the model or config file from command
+line, trains and evaluates ensemble of anomaly models to get quantitative and qualitative
+results.
+"""
+
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import logging
+import warnings
+from argparse import ArgumentParser, Namespace
+from itertools import product
+
+from pytorch_lightning import Trainer, seed_everything
+
+from anomalib.config import get_configurable_parameters
+from anomalib.models import get_model
+from anomalib.utils.loggers import configure_logger, get_experiment_logger
+from anomalib.models.ensemble import (
+    get_ensemble_datamodule,
+    get_ensemble_callbacks,
+    prepare_ensemble_configurable_parameters,
+    get_prediction_storage,
+    EnsembleTiler,
+    post_process,
+    log_metrics,
+)
+
+logger = logging.getLogger("anomalib")
+
+
+def get_parser() -> ArgumentParser:
+    """Get parser.
+
+    Returns:
+        ArgumentParser: The parser object.
+    """
+    parser = ArgumentParser()
+    parser.add_argument("--model", type=str, default="padim", help="Name of the algorithm to train/test")
+    parser.add_argument("--config", type=str, required=False, help="Path to a model config file")
+    parser.add_argument("--ens_config", type=str, required=True, help="Path to an ensemble configuration file")
```

Comment:
Yes, that was one thing we really quickly discussed last time with Samet. It can easily be done, and will I think even simplify setting up the config for users. So this should all be part of base config? I assume optional but documented that it should be added inside?